src/containers/NoAccessPage/NoAccessPage.test.js

```diff
@@ -11,10 +11,10 @@ const { screen, userEvent } = testingLibrary;
 const noop = () => null;
 
 describe('NoAccessPageComponent', () => {
-  it('Check that /no-posting-right has heading and content', () => {
+  it('Check that /no-posting-rights has heading and content', () => {
```

Comment:
Perhaps, you could add another test for the /no-transaction-rights

Reply:
Test added!